providers/implementations/ciphers/cipher_cts.c

```diff
@@ -46,7 +46,7 @@
  *      Otherwise it is the same as CS2.
  */
 
-#include "internal/e_os.h" /* strcasecmp */
+#include "internal/e_os.h" /* OPENSSL_strcasecmp */
```

Comment:
Wrong change - should be deleted including the #include.

Reply:
Done, thanks